### DISCUSSION TITLE
Please implement an "udp-ping" as check, too

### DISCUSSION THREAD

**Discussion by thomasmerz** [upptime/upptime#276] [Ideas]
Regarding https://github.com/upptime/upptime/issues/273 and https://github.com/upptime/upptime/discussions/204: could you please also implement an **udp-ping** as check because some services are UDP-only like Wireguard?

**Comment by wolfier74** [comment]
Good idea

**Comment by AnandChowdhary** [comment]
Can you find an npm package like `tcp-ping` that has this function?

**Reply by thomasmerz** [reply]
What about these?
* https://www.npmjs.com/package/udping
* https://github.com/danielzzz/node-ping
* https://snyk.io/advisor/npm-package/raw-socket
* https://nodejs.org/api/dgram.html
🤔  (sorry, I'm "only" a sysadmin, no real programmer…)

**Reply by thomasmerz** [reply]
@AnandChowdhary , do you have found the time yet to think about this feature? 🤔

**Reply by thomasmerz** [reply]
@AnandChowdhary , can you please give a short answer regarding this feature request? Thanks a lot.

**Reply by AnandChowdhary** [reply]
Happy to accept a PR!

**Comment by raykai** [comment]
would love to have this udp ping option for monitoring self hosted games servers.
